src/command/handlers/dig/shared.ts

```diff
@@ -35,10 +35,10 @@ export const IP_REG_EXP = ipRegex();
 export const SharedDigParser = {
 	parseSection (values: string[]): DnsSection {
 		return {
-			name: values[0],
-			type: values[3],
+			name: values[0] ?? null,
+			type: values[3] ?? null,
 			ttl: Number(values[1] ?? ''),
-			class: values[2],
+			class: values[2] ?? null,
```

Comment:
I wonder if you found a specific case where the values are missing, and the nulls are used? Our API docs currently say the fields are always present and not null, so I wonder if we should change that.

Reply:
I don't have such a case, so if we are sure values are always there we can replace it with `values[0]!`.

Reply:
I don't think it can happen unless there's a bug in the parser or invalid dig output so `!` seems better.